addons/mass_mailing/models/mailing.py

```diff
@@ -344,7 +344,7 @@ def _compute_next_departure(self):
 
     @api.depends('email_from', 'mail_server_id')
     def _compute_warning_message(self):
-        for mailing in self:
+        for mailing in self.filtered(lambda mailing: mailing.mailing_type == "mail"):
```

Comment:
Better to initialize for have a value, like
```
    self.warning_message = False
    for mailing in ...
```

Reply:
Thanks @tde-banana-odoo done